README.rst

```diff
@@ -610,6 +610,13 @@ by disabling both web and email option for a notification setting.
 then ``email_notification`` or ``web_notification`` option of concerned notification type will be used
 respectively.
 
+Deleting Notification Preferences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Deleting the notification preferences is an advanced option. Users should turn off web and email
+notifications instead of deleting notification preferences. Deleted notification preferences
+might get re-created by automation.
```

Comment:
what about "may be re-created automatically if the system needs it."? Or something along this line.